pkg/annotations/constants.go

```diff
@@ -45,6 +45,7 @@ const (
 	IngressSuffixAuthSessionCookie            = "auth-session-cookie"
 	IngressSuffixAuthSessionTimeout           = "auth-session-timeout"
 	IngressSuffixTargetNodeLabels             = "target-node-labels"
+	IngressSuffixManageSecurityGroupRules     = "manage-security-group-rules"
```

Comment:
wondering whether it would be better to be specific like `manage-node-security-group-rules` or `manage-backend-security-group-rules`